cloudferrylib/os/actions/convert_compute_to_image.py

```diff
@@ -67,6 +66,13 @@ def run(self, info=None, **kwargs):
                     images_body[image_id] = {utl.IMAGE_BODY: {},
                                              utl.META_INFO: {
                                              utl.INSTANCE_BODY: [instance]}}
+                    try:
+                        image_resource.get_ref_image(image_id)
```

Comment:
We don't read a body of the image at this step, but we should test this situation anyway.